src/Components/CreateImageWizard/steps/Snapshot/Snapshot.tsx

```diff
@@ -85,9 +87,17 @@ export default function Snapshot() {
                 ]}
                 onChange={(_, val) => dispatch(changeSnapshotDate(val))}
               />
+
               <Button
                 variant="link"
-                onClick={() => dispatch(changeSnapshotDate(''))}
+                onClick={async () => {
+                  //Patternfly DatePicker seems to only clear error text if value is reset to '',
+                  // if you have an invalid date (2000-01-010000) and try to reset it, it must be set to '' first
+                  dispatch(changeSnapshotDate(''));
+                  setTimeout(() => {
+                    dispatch(changeSnapshotDate(yyyyMMddFormat(new Date())));
+                  }, 1);
```

Comment:
I wanted to highlight this.  i would thought that just 
```
                    dispatch(changeSnapshotDate(yyyyMMddFormat(new Date())));
```
would work, but if you type in something like  '2024-01-010000'  and then click reset, it won't actually reset to the current date.  By resetting to '' and then resetting to the date, it works fine.